### PR TITLE
release: v1.7.2

### PR DESCRIPTION
**Pull request by raelga**
Initial release of the `cert-manager-helm` operator, with v1.7.2 helm chart.

/kind release
/priority important-longterm
/assign

### CODE REVIEW

**Comment by slopezz** [comment]
Nice work!

I would do a few cosmetic changes like we did with external-secrets meta-operator after a team discussion:
- **Security change** Don't ship the real CRDs with the operator image https://github.com/external-secrets/external-secrets-helm-operator/blob/main/hack/download-helm-chart.sh#L17 (so remove them prior to meta operator image creation), as a safe measure in case someone sets by mistake `installCRDs: true` (only OLM can manage CRDs)
- **Cosmetic change**: Use the helm-chart directory [helm-charts/cert-manager](https://github.com/3scale-ops/cert-manager-helm-operator/tree/release/v1.7.2/helm-charts/cert-manager) as a dynamic directory only while creating the meta operator image, but don't push the operator templates to Git Hub because they are irrelevant to us (we are just encapsulating the templates in the meta operator image). So the same we did at https://github.com/external-secrets/external-secrets-helm-operator/tree/main/helm-charts/external-secrets. The idea for future PRs where we update operator image, is that we will just need mostly to update a single variable with the upstream operator version

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+		--untar --untardir ${HELM_CHARTS_PATH} \
+		--version=$(VERSION) jetstack/cert-manager
+	@sed -i "s|installCRDs: [^ ]*|installCRDs: false|g" \
+		$(HELM_CHARTS_PATH)/cert-manager/values.yaml
+
+TMP_FOLDER="$(PWD)/tmp"
+
+render-cdrs-from-helm-chart: helm download-helm-chart ## Generates the CRD definitions from the existing helm chart
+	@test ! -d $(TMP_FOLDER)/render || find $(TMP_FOLDER)/render -delete;
+	@$(HELM) template --set installCRDs=true --include-crds $(HELM_CHARTS_PATH)/cert-manager \
```
Comment (slopezz) [inline Makefile:281]: Why set `installCRDs=true` when in the previous  **download-helm-chart** target you are already doing a safe **sed** with the correct value `false`?

Comment (raelga) [reply Makefile:281]: This step is only used to generate the CRDs, in a temporary folder, that's why that flag is required here but not in the download step.

File: bundle.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
+FROM scratch
+
+# Core bundle labels.
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=cert-manager-helm-operator
+LABEL operators.operatorframework.io.bundle.channels.v1=alpha
```
Comment (slopezz) [inline bundle.Dockerfile]: It is configured an alpha release, but the release that will be created by the github action once this PR gets merged is a stable, right?

Maybe the prepare-stable-release target should be run?

**Pull request merged**